src/components/Accordion/useAccordionState.js

```diff
@@ -0,0 +1,12 @@
+import { useState } from 'react'
+
+export const useAccordionState = (initialState) => {
```

Comment:
Just to make sure I'm understanding everything, is `useAccordionState` only meant to be used in the example to show how `onToggle` and `expanded` props work?

Reply:
yeah, the consumer can set everything up with something like:
`const { expanded, onToggle } = useAccordionState({ 1: true })`